### ISSUE TITLE
DNDB - Bode's Belt of Cloud Giant Strength is too generous

### ISSUE THREAD

**Issue by trey-kirk-sp** [mr-ice/maptool-macros#89]
When equipped, strength is built as 72 or some shit like that.

**Comment by trey-kirk-sp** [comment]
Early DTO written before Dummy knew how to properly read granted modifiers is reading this:
```
{
                    "availableToMulticlass": true,
                    "bonusTypes": [],
                    "dice": null,
                    "duration": null,
                    "entityId": 1,
                    "entityTypeId": 1472902489,
                    "fixedValue": 27,
                    "friendlySubtypeName": "Strength Score",
                    "friendlyTypeName": "Set",
                    "id": "408",
                    "isGranted": true,
                    "modifierSubTypeId": 175,
                    "modifierTypeId": 9,
                    "requiresAttunement": true,
                    "restriction": "",
                    "statId": null,
                    "subType": "strength-score",
                    "type": "set",
                    "value": 27,
                    "componentId": 4829,
                    "componentTypeId": 112130694
                }
```
a. twice (because it shows up in data.inventory and data.grantedModifiers)
b. as a bonus instead of setting it

Need to refactor getAbilities to work more like later DTOs like Skills did when determining granted mods.

**Comment by trey-kirk-sp** [comment]
Fixed in #31. Need to update one of the Live Test characters with the belt for regression testing.

**Comment by trey-kirk-sp** [comment]
so good
